eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOconsoleTest.java

```diff
@@ -47,13 +48,19 @@
  */
 @SuppressWarnings("JTCOP.RuleAllTestsHaveProductionClass")
 final class EOconsoleTest {
+
+    /**
+     * Hello.
+     */
+    private static final String HELLO = "Hello";
```

Comment:
@c71n93 this is not a good idea in general - to share literals between tests. Can't we use a different string in each test?